partner_report_open_invoices/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Partner Report Open Invoices
+============================
+
+This module will create a report that contains the details of open invoices for partner in partner form view.
+
+
+Installation
+============
+
+This module depends on:
+
+* account
+
+Configuration
+=============
+
+* There needs a configuration of a message which will appear under Company Settings, just like Overdue Message.
+
+Usage
+=====
+
+* The module generates a report on partner which contains a list of outstanding journal entries and at the end it will
+show up the invoice messages configured inside the Company Configuration.
+
```

Comment:
Please also add the 'Try me on Runbot' link, see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst as reference.